test/accountSpec.js

```diff
@@ -93,6 +116,7 @@ describe('Accounts', function () {
             connector: 'http://localhost:4321'
           }
         ])
+        .expect(validAccounts)
```

Comment:
We could do something like
```
.expect(res => {
  expect(validAccounts.bind(validAccounts, res)).to.not.throw(Error)
 })
```
But that's an extra two lines of code each time we want to validate a schema, and in my opinion less readable. I'm still searching for an ideal solution.